src/openrct2/rct2/S6Exporter.cpp

```diff
@@ -996,6 +996,8 @@ void S6Exporter::ExportSpriteCommonProperties(RCT12SpriteBase* dst, const Sprite
 
 void S6Exporter::ExportSpriteVehicle(RCT2SpriteVehicle* dst, const Vehicle* src)
 {
+    auto ride = get_ride(src->ride);
```

Comment:
Should this be `const auto&` like the other places?

Reply:
If at all it should be ```const auto*```